netlify.toml

```diff
@@ -7,7 +7,7 @@ NODE_ENV = "production"
 HUGO_VERSION = "0.141.0"
 HUGO_ENABLEGITINFO = "true"
 HUGO_ENVIRONMENT = "preview"
-SECRETS_SCAN_OMIT_PATHS = "public/contribute/file-conventions/index.html"
+SECRETS_SCAN_OMIT_PATHS = "public/contribute/file-conventions/index.html,public/content/manuals/build/building/cdi.html"
```

Comment:
```suggestion
SECRETS_SCAN_OMIT_PATHS = "public/contribute/file-conventions/index.html,public/build/building/cdi/index.html"
```

Reply:
Yes I figured `public/content/manuals/build/building/cdi.html` was not correct from https://github.com/docker/docs/pull/23069#issuecomment-3062188495 looking at the logs :sweat_smile: